documentation/source/index.rst

```diff
@@ -0,0 +1,20 @@
+.. PSYDAC documentation documentation master file, created by
+   sphinx-quickstart on Fri May 12 13:11:39 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to PSYDAC documentation's documentation!
```

Comment:
ah yes, this happened because sphinx-quickstart asked me for a project name (and author, version, …) without a priori telling me how this information would be used in the index.rst file.

I can change that later tonight, but I intend on deleting the whole /documentation folder anyways. It’s just there so we can add the sphinx.yml file to the devel branch without tests failing and it will be replaced by a /docs folder with contains the actual documentation.

Reply:
There is no rush, but it should be corrected because it will be deployed to the website after the merge.

Reply:
d3be4e7ff2764d5c2f8bb0a9bb0e4a678c59bda7

Reply:
Copyright and authors information should not be hardcoded in the documentation. Please point to the `LICENSE` and `AUTHORS` files instead. Possibly some metadata could be extracted from the `pyproject.toml` file.